tests/client.py

```diff
@@ -822,6 +823,10 @@ def create_contacts(self):
             ]
         }
 
+        #Get the current time in seconds
+        date= datetime.datetime.utcnow() - datetime.datetime(1970, 1, 1)
+        seconds =(date.total_seconds())
```

Comment:
```suggestion     
        # Get the current time in seconds
        date = datetime.datetime.utcnow()
        seconds = datetime.datetime.timestamp(date)
```
This would only give you precision to the second. We probably want to go to millisecond or microsecond precision as this suggestion would do

Reply:
Changed the time to milliseconds as suggested